_security/configuration/best-practices.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: default
+title: Best Practices
+parent: Best Practices
+nav_order: 11
+---
+
+# Best Practices for OpenSearch Security plugin
+
+Setting up secure practices in OpenSearch is crucial for protecting your data. Here are our top 10 guidelines that offer clear steps on keeping your system safe.
+
+## 1. Use your own PKI (public key infrastructure) to setup SSL/TLS for OpenSearch
+
+Although it requires a bit more effort at the beginning, it provides you with all the flexibility needed to set up SSL/TLS in the most secure and performant way.
+
+### Enable SSL/TLS for node and REST layer traffic
+
+The SSL/TLS is enabled by default for transport layer which is used by node to node communication. The SSL/TLS on REST layer is disabled by default.
+
+The following setting is required to enable encryption at REST layer: `plugins.security.ssl.http.enabled: true`
+
+For additional configuration options, such as specifying certificate paths, keys and certificate-authority files you can refer the steps on [configuring TLS certificates]({{site.url}}{{site.baseurl}}/security/configuration/tls/)
+
+### Replace all demo certificates with your own PKI
+The certificates generated when initializing OpenSearch cluster with `install_demo_configuration.sh` are not suitable for production. These should be replaced with your own certificates.
+
+You can generate custom certificates in a few different ways. One approach is by using OpenSSL, described in detail on [Generating self-signed certificates]({{site.url}}{{site.baseurl}}/security/configuration/generate-certificates/), alternatively there are tools available online which can simplify the certificate creation process.
```

Comment:
Do we have any other approaches we can document for demo certificates, since we say "a few different ways?"

Reply:
+1. Additionally, if we reference online tools, we should reference at least a couple.